libr/crypto/p/crypto_aes_wrap.c

```diff
@@ -60,16 +55,15 @@ static bool update(RCryptoJob *cj, const ut8 *buf, int len) {
 		return false;
 	}
 
-	ut8 *const obuf = calloc (1, len + BLOCK_SIZE);
+	ut8 *const obuf = calloc (1, len + AES_WRAP_BLOCK_SIZE);
 	if (!obuf) {
 		return false;
 	}
-	long *obuf_ptr = (long *)obuf;
 
 	if (!cj->iv) {
-		cj->iv = malloc (BLOCK_SIZE);
+		cj->iv = malloc (AES_WRAP_BLOCK_SIZE);
 		if (cj->iv) {
```

Comment:
ive added a new dev rule that mallocs with constant/fixed size dont require an if check, heap allocations are never sueposed to fail unless the value is really large, zero or negative. we can reduce a lot of checks by just avoiding that without introducing errors.